app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryPreferencesEducator.tsx

```diff
@@ -0,0 +1,195 @@
+import { Heading, t } from "@bloom-housing/ui-components"
+import type { RailsLotteryResult } from "../../api/types/rails/listings/RailsLotteryResult"
+import { defaultIfNotTranslated, renderMarkup } from "../../util/languageUtil"
+import { preferenceNameHasVeteran } from "../../util/listingUtil"
+
+export interface ListingDetailsLotteryPreferencesProps {
+  lotteryBucketsDetails: RailsLotteryResult
+  isEducatorOne: boolean
+}
+interface PreferencesProps {
+  unitsAvailable: number
+  totalSubmittedApps: number
+  preferenceName: string
+  numVeteranApps: number
+  lastPref: boolean
+}
+
+const LotteryPreferences = ({
+  unitsAvailable,
+  totalSubmittedApps,
+  preferenceName,
+  numVeteranApps,
+  lastPref,
+}: PreferencesProps) => (
+  <div>
+    <div className="px-8">
+      <Heading className="font-sans font-semibold text-xs tracking-wide uppercase" priority={3}>
+        {defaultIfNotTranslated(
+          `listings.lotteryPreference.${preferenceName}.title`,
+          preferenceName
+        )}
+      </Heading>
+      <p className="text-sm">{t("lottery.upToXUnitsAvailable", { units: unitsAvailable ?? 0 })}</p>
+      {numVeteranApps ? (
+        <>
+          <p className="text-gray-700 text-sm">
+            {t("lottery.numberApplicantsQualifiedForPreference.veteran", {
+              number: numVeteranApps,
+              plural: numVeteranApps > 1 ? "s" : "",
+            })}
+          </p>
+          <p className="text-gray-700 text-sm">
+            {t("lottery.numberApplicantsQualifiedForPreference.nonVeteran", {
+              number: totalSubmittedApps,
+              plural: numVeteranApps > 1 ? "s" : "",
+            })}
+          </p>
+        </>
+      ) : (
+        <p className="text-gray-700 text-sm">
+          {t("lottery.numberApplicantsQualifiedForPreference", {
+            number: totalSubmittedApps,
+            plural: numVeteranApps > 1 ? "s" : "",
+          })}
+        </p>
+      )}
+    </div>
+    <hr className={lastPref ? "mt-4" : "my-4"} />
+  </div>
+)
+
+export const ListingDetailsLotteryPreferencesEducator = ({
+  lotteryBucketsDetails,
+  isEducatorOne,
+}: ListingDetailsLotteryPreferencesProps) => {
+  const veteranAppsByPreferenceMap = {}
+  lotteryBucketsDetails.lotteryBuckets
+    .filter((lotteryBucket) => preferenceNameHasVeteran(lotteryBucket.preferenceName))
+    .forEach((lotteryBucket) => {
+      veteranAppsByPreferenceMap[lotteryBucket.preferenceShortCode] =
+        lotteryBucket.totalSubmittedApps
+    })
+
+  return (
+    <div className="text-sm">
+      <header className="border-b border-gray-450 mb-4">
+        <Heading styleType="underlineWeighted" className="mx-8" priority={2}>
+          {t("lottery.housingPreferences.educator")}
+        </Heading>
+        <div className="pb-4 text-gray-700 text-xs mx-8 translate">
+          {renderMarkup(
+            `${t("lottery.rankingOrderNote", {
+              lotteryRankingVideoUrl: "https://www.youtube.com/watch?v=4ZB35gagUl8",
```

Comment:
suggestion: URLs should be constants imported from `app/javascript/modules/constants.tsx`